src/coreclr/jit/compiler.cpp

```diff
@@ -5055,6 +5055,9 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
     // Expand thread local access
     DoPhase(this, PHASE_EXPAND_TLS, &Compiler::fgExpandThreadLocalAccess);
 
+    // Expand casts
+    DoPhase(this, PHASE_EXPAND_CASTS, &Compiler::fgLateCastExpansion);
```

Comment:
It seems like we should combine all these "expand helper" IR traversals into one single IR traversal at some point.

Reply:
Agree. I am going to make some clean up with the SplitBlock part separately in all of them